assets/src/edit-story/masks/index.js

```diff
@@ -100,6 +102,12 @@ function WithtMask({ elementId, mask, fill, style, children, ...rest }) {
     ...style,
   };
 
+  const transformFlip = getTransformFlip(flip);
+  if (transformFlip) {
```

Comment:
Using this here seems to be breaking the panning for a flipped image, probably will still have to create a separate flip wrapper... Or add a prop, sth like "applyFlip" to exclude using it in edit mode.